ffac-autoupdater-wifi-fallback/luasrc/usr/sbin/autoupdater-wifi-fallback

```diff
@@ -97,22 +97,20 @@ end
 
 local function switch_to_fallback_mode(radio, ssid, bssid)
   autil.log('out', 'connecting to '  .. radio .. ' ' .. ssid .. ' ' .. bssid)
-	os.execute('sleep 2')
-	ubus:call('network', 'add_dynamic', {name = "fallback", ifname = "fallback", proto = "dhcp", device = "fallback", l3_device = "fallback"})
-	ubus:call('network', 'add_dynamic', {name = "fallback6", ifname = "fallback", proto = "dhcpv6", device = "fallback", l3_device = "fallback"})
-	ubus:call('network.interface.fallback', 'add_device', {name = "fallback"})
-	ubus:call('network.interface.fallback6', 'add_device', {name = "fallback"})
-	ubus:call('network.interface.fallback', 'up', {})
-	ubus:call('network.interface.fallback6', 'up', {})
```

Comment:
are you really sure about not needing those lines? isn't it more clean?

Reply:
There is no change from them, as the link is already up, but there is no network yet to connect to.
When running `ubus call network.interface.fallback6 status`, one can see that the command does not change something.
Generally it looks like:

```
{
	"up": false,
	"pending": false,
	"available": false,
	"autostart": true,
	"dynamic": false,
	"proto": "dhcpv6",
	"data": {
		
	},
	"errors": [
		{
			"subsystem": "interface",
			"code": "NO_DEVICE"
		}
	]
}
```

and as autostart is set, there is no need to run the up command after running `iw dev fallback connect`